.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
```

Comment:
Yeah, this is something that my IDE seems to add to all my projects. I'm afraid I'm not at all sure what they do but I'm not aware of them causing any issues.